src/kraken-net/Model/OptimizeRequestBase.cs

```diff
@@ -48,5 +48,8 @@ public SamplingScheme SamplingScheme
 
         [JsonProperty("dev")]
         public bool Dev { get; set; }
+
+        [JsonProperty("quality")]
+        public int Quality { get; set; }
```

Comment:
This should be optional and only included within the JSON message when a value has been provided.